artifactory/cli.go

```diff
@@ -711,6 +719,7 @@ func getDownloadFlags() []cli.Flag {
 		},
 		getIncludeDirsFlag(),
 		getPropertiesFlag("Only artifacts with these properties will be downloaded."),
+		getExcludePropertiesFlag("Only artifacts without the specified value for the property will be downloaded"),
```

Comment:
I suggest changing the message to the below, to be consistent with  getPropertiesFlag* message:
**Only artifacts without the specified properties will be downloaded.**
Notice that this should be changed in multiple places in this file.